To: vim_dev@googlegroups.com
Subject: Patch 7.3.082
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.082
Problem:    Leaking file descriptor when hostname doesn't exist.
Solution:   Remove old debugging lines.
Files:	    src/netbeans.c


*** ../vim-7.3.081/src/netbeans.c	2010-12-08 13:11:15.000000000 +0100
--- src/netbeans.c	2010-12-17 12:13:32.000000000 +0100
***************
*** 323,334 ****
      server.sin_port = htons(port);
      if ((host = gethostbyname(hostname)) == NULL)
      {
- 	if (mch_access(hostname, R_OK) >= 0)
- 	{
- 	    /* DEBUG: input file */
- 	    sd = mch_open(hostname, O_RDONLY, 0);
- 	    goto theend;
- 	}
  	nbdebug(("error in gethostbyname() in netbeans_connect()\n"));
  	PERROR("gethostbyname() in netbeans_connect()");
  	goto theend;
--- 323,328 ----
*** ../vim-7.3.081/src/version.c	2010-12-08 19:56:52.000000000 +0100
--- src/version.c	2010-12-17 12:17:11.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     82,
  /**/

-- 
How To Keep A Healthy Level Of Insanity:
1. At lunch time, sit in your parked car with sunglasses on and point
   a hair dryer at passing cars. See if they slow down.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
